_i18n/de/resources/moneropedia/account.md

```diff
@@ -1,34 +1,33 @@
 ---
-terms: ["account", "accounts", "wallet", "wallets", "Account", "Accounts"]
-summary: "similar in function to a bank account, contains all of your sent and received transactions"
+terms: ["account", "accounts", "wallet", "wallets", "Konto", "Konten", "Kontos", "Wallet", "Wallets"]
+summary: "Funktionsähnlich zu einem Bankkonto, enthält all deine gesendeten und empfangenen Transaktionen"
 ---
 
-{% include untranslated.html %}
-### The Basics
+### Grundlagen
 
-Those familiar with Monero's predecessors will be more familiar with the term *wallet* to describe this. In Monero we call this an account, and it is a private account owned and operated by a Monero user.
+Für jene, die mit Moneros Vorgängern vertraut sind, wird zur Beschreibung der Begriff *Wallet* geläufiger sein. Bei Monero nennen wir dies ein Konto, welches ein privates, von einem Monero-Nutzer geführtes und in dessen Besitz befindliches Konto ist.
 
-Your account contains all of the Monero @transactions you have sent and received. Your account balance is a sum of all the Monero you've received, less the Monero you've sent. When using Monero you may notice that your account has two balances, a locked and an unlocked balance. The unlocked balance contains funds that can be spent immediately, and the locked balance contains funds that you can't spend right now. You may receive a transaction that has an @unlock-time set, or you may have sent some Monero and are waiting for the @change to come back to your wallet, both situations that could lead to those funds being locked for a time.
+Dein Konto enthält die @Transaktionen aller Monero, welche du gesendet und empfangen hast. Dein Kontoguthaben ist die Summe aller Monero, die du empfangen hast, abzüglich derer, die du gesendet hast. Bei der Nutzung von Monero kannst du feststellen, dass dein Konto zwei Guthaben hat: ein gesperrtes und ein offenes. Das offene Guthaben enthält Geldmittel, die unmittelbar ausgegeben werden können, das gesperrte Guthaben enthält jene, die nicht direkt ausgabebereit sind. Letzteres könnte beispielsweise daran liegen, dass eine von dir empfangene Transaktion eine gesetzte @Freigabedauer hat, oder dass du noch auf @Wechselgeld der von dir gesendeten Moneros wartest - beide Situationen könnten dazu führen, dass dieses Guthaben für eine Weile gesperrt ist.
 
-A key difference between traditional electronic currency and Monero is that your account resides only under your control, normally on your computer, and cannot be accessed by anyone else if you [practice good security](#practicing-good-security).
+Ein entscheidender Unterschied zwischen traditioneller elektronischer Währung und Monero ist, dass dein Konto, das im Normalfall auf deinem Computer liegt, einzig und allein in deiner Kontrolle ist und nicht von anderen abgerufen werden kann, sofern du eine [gute Sicherheit betreibst] (#Gute-Sicherheit-betreiben).
```

Comment:
```suggestion
Ein entscheidender Unterschied zwischen traditioneller elektronischer Währung und Monero ist, dass dein Konto, das im Normalfall auf deinem Computer liegt, einzig und allein in deiner Kontrolle ist und nicht von anderen abgerufen werden kann, sofern du eine [gute Sicherheit betreibst](#Gute-Sicherheit-betreiben).
```
This is the structure of a markdown link. There must be no space between `]` and `)` for a link to work correctly (see the original version). Please check other links in the other files as well.